spec/tictactoe/ruby_spec.rb

```diff
@@ -1,6 +1,24 @@
-RSpec.describe Tictactoe do
+require "./lib/tictactoe/game"
+RSpec.describe Tictactoe::Game, Tictactoe::Board do
 
-  it "should add 1 to 1" do
-    expect(1+1).to eq(2)
+  context "when both players play" do
+    xit "should signal a draw when there is no winner" do
+
+      game_positions = Array.new(9,"-")
+      board = Tictactoe::Board.new(game_positions)
+      new_game = Tictactoe::Game.new(board)
+      player_one_moves = [1,3,5,8]
+      player_two_moves = [2,4,6,7,9]
+
+      for p1_moves in player_one_moves
```

Comment:
In Ruby, it is more idiomatic to use `each` rather than for loop. For example,

```rb
player_one_moves.each do |move|
  new_game.play(move, 'X')
end
```